src/familylink/client.py

```diff
@@ -37,15 +38,23 @@ def __init__(
         """
         self.account_id = account_id
 
-        cookie_kwargs = {}
-        if cookie_file_path:
-            if not cookie_file_path.exists():
-                raise ValueError(f"Cookie file not found: {cookie_file_path}")
-            if not cookie_file_path.is_file():
-                raise ValueError(f"Cookie file is not a file: {cookie_file_path}")
-            cookie_kwargs["cookie_file"] = str(cookie_file_path.resolve())
+        if browser == "txt":
+            if not cookie_file_path:
+                cookie_file_path = "./cookies.txt"
 
-        self._cookies = getattr(browser_cookie3, browser)(**cookie_kwargs)
+            cookie_path = Path(cookie_file_path)
+            self._cookies = MozillaCookieJar(cookie_path)
+            self._cookies.load()
+        else:
+            cookie_kwargs = {}
+            if cookie_file_path:
+                if not cookie_file_path.exists():
+                    raise ValueError(f"Cookie file not found: {cookie_file_path}")
+                if not cookie_file_path.is_file():
+                    raise ValueError(f"Cookie file is not a file: {cookie_file_path}")
+                cookie_kwargs["cookie_file"] = str(cookie_file_path.resolve())
+
+                self._cookies = getattr(browser_cookie3, browser)(**cookie_kwargs)
```

Comment:
```suggestion
            self._cookies = getattr(browser_cookie3, browser)(**cookie_kwargs)
```
This line should be dedented